src/editor/toolbox.ts

```diff
@@ -141,3 +141,99 @@ export class ToolboxButton {
         return new ToolboxButton(obj.text, obj?.id);
     }
 }
+
+/**
+ * Create the cascaded menu div object and its options along with their action handlers.
+ */
+function constructCascadedMenuObj(
```

Comment:
This will allow us to create cascaded menus on any dom object. Previously this functionality was coupled with variable reference buttons.